### PR TITLE
feat(fetch): support FormData as multipart parameter

### PR DESCRIPTION
**Pull request by yury-s**
Reference https://github.com/microsoft/playwright/issues/28070

### CODE REVIEW

File: docs/src/api/params.md [hunk 1]
```diff
@@ -405,7 +405,7 @@ An instance of [FormData] can be created via [`method: APIRequestContext.createF
 
 ## js-python-fetch-option-multipart
 * langs: js, python
-- `multipart` <[Object]<[string], [string]|[float]|[boolean]|[ReadStream]|[Object]>>
+- `multipart` <[FormData]|[Object]<[string], [string]|[float]|[boolean]|[ReadStream]|[Object]>>
```
Comment (dgozman) [inline docs/src/api/params.md:408]: - Is `FormData` available in python?
- We will need a custom link on `playwright.dev` leading to MDN.

File: docs/src/api/class-formdata.md [hunk 2]
```diff
@@ -14,6 +14,77 @@ FormData form = FormData.create()
...
 
+## method: FormData.append
+* since: v1.43
+- returns: <[FormData]>
+
+Appends a new value onto an existing key inside a FormData object, or adds the key if it
+does not already exist. File values can be passed either as `Path` or as `FilePayload`.
+Multiple fields with the same name can be added.
+
+The difference between set() and append() is that if the specified key already exists,
```
Comment (dgozman) [inline docs/src/api/class-formdata.md:25]:
```suggestion
The difference between [`method: FormData.set`] and [`method: FormData.append`] is that if the specified key already exists,
```

File: docs/src/api/class-formdata.md [hunk 3]
```diff
@@ -14,6 +14,77 @@ FormData form = FormData.create()
...
+set() will overwrite all existing values with the new one, whereas append() will append
```
Comment (dgozman) [inline docs/src/api/class-formdata.md:26]:
```suggestion
setting will overwrite all existing values with the new one, whereas appending will add
```

**Review by dgozman — APPROVED** [review]

File: docs/src/api/params.md (hunk 1 above)
Comment (yury-s) [reply docs/src/api/params.md:408]: > Is FormData available in python?

No, only csharp and java.

File: docs/src/api/class-formdata.md (hunk 2 above)
Comment (yury-s) [reply docs/src/api/class-formdata.md:25]: These links will not work in js and python.

File: docs/src/api/params.md (hunk 1 above)
Comment (yury-s) [reply docs/src/api/params.md:408]: https://github.com/microsoft/playwright.dev/pull/1360 adds FormData link

**Review by dgozman — APPROVED** [review]

**Pull request merged**